Library/Homebrew/cask/lib/hbc/cli/abstract_command.rb

```diff
@@ -7,11 +7,12 @@ class AbstractCommand
       include Options
       include Homebrew::Search
 
-      option "--[no-]binaries", :binaries,      true
-      option "--debug",         :debug,         false
-      option "--verbose",       :verbose,       false
-      option "--outdated",      :outdated_only, false
-      option "--require-sha",   :require_sha,   false
+      option "--[no-]binaries",   :binaries,      true
+      option "--debug",           :debug,         false
+      option "--verbose",         :verbose,       false
+      option "--outdated",        :outdated_only, false
+      option "--require-sha",     :require_sha,   false
+      option "--[no-]quarantine", :quarantine,    true
```

Comment:
I know this PR has been nitpicked to pieces but can you please add a line for the `--no-quarantine` option to `Library/Homebrew/manpages/brew-cask.1.md`?

Reply:
No problem! Let me know if the next commit is OK for you.